lib/asciidoctor/converter/manpage.rb

```diff
@@ -51,6 +51,12 @@ def manify str, opts = {}
       append_newline ? %(#{str}#{LF}) : str
     end
 
+    def preserve_backslash str
+      str = str.
+        gsub(/\\/, '\\e')
```

Comment:
Could you reduce this to one line and change the first argument to a single quoted backslash string?

Perhaps we can add this as an option named `:preserve_backslash` to the manify method. That way, we don't need an extra method in the first two cases. Obviously, we still need it for the inline case.